CONTRIBUTING.md

```diff
@@ -88,10 +88,9 @@ and are very interested in folks joining in to help us with it.
 
 ### Help address bugs
 
-As Carbon's design and eventually implementation begin to take shape, we'll
+As Carbon's design and eventually implementation begins to take shape, we'll
 inevitably end up with plenty of bugs. Helping us triage, analyze, and address
-them is always a great way to get involved. When we have the bug tracker(s) set
-up for this, we'll update this section with ideas of how to find these and get
+them is always a great way to get involved. When we have the bug tracker(s) set up for this, we'll update this section with ideas on how to find these and get
```

Comment:
please run prettifier on this

Reply:
like how? is it an extension?

Reply:
https://prettier.io/ is the tool we use.  The exact how depends on what system you are running on.